crypto/rsa/rsa_sp800_56b_check.c

```diff
@@ -324,7 +329,8 @@ int ossl_rsa_sp800_56b_check_public(const RSA *rsa)
         goto err;
     }
 
-    ret = ossl_bn_miller_rabin_is_prime(rsa->n, 0, ctx, NULL, 1, &status);
+    /* Number of MR rounds 5 from FIPS 186-5 Table B.1 - the highest one */
+    ret = ossl_bn_miller_rabin_is_prime(rsa->n, 5, ctx, NULL, 1, &status);
```

Comment:
Could the comment be updated to mention the particular section from FIPS 186-5 where that table is found?  i.e. something like this:

```
/* Use 5 Enhanced-MR rounds.  This is the highest number of rounds recommended in FIPS 186-5, Section B.3, Table B.1 */
```

Reply:
And from that table, if the length of the modulus we are checking is less than 1024 bits, then 5 rounds might not be sufficient.  But perhaps we don't need to worry about that case.

Reply:
@jamuir This table is used only as kind of arbitrary guidance, its purpose is very much different and it would be probably OK to use even much lower values for this particular test. The check is to reject primes here not to accept only primes. At worst an artificially created valid public key is rejected as invalid.

Reply:
I think the comment is sufficient as is.

Reply:
Note that we have an BN_prime_checks_for_size() function for this in bn.h, which you could pass nbits/2.

Reply:
> I think the comment is sufficient as is.

I disagree.  Referencing the section from FIPS 186-5 where the table is found would have saved me a few minutes.

Reply:
Comment updated.